src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkArtistMarket.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Column,
+  DecreaseIcon,
+  Flex,
+  GridColumns,
+  IncreaseIcon,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Media } from "Utils/Responsive"
+import { MyCollectionArtworkArtistMarket_marketPriceInsights } from "../../../../../__generated__/MyCollectionArtworkArtistMarket_marketPriceInsights.graphql"
+
+export const MyCollectionArtworkArtistMarket = ({
+  marketPriceInsights,
+}: {
+  marketPriceInsights: MyCollectionArtworkArtistMarket_marketPriceInsights
+}) => {
+  const renderInsight = (name: string, value: string) => {
+    return (
+      <Column span={[6, 4, 2]}>
+        <Flex flexDirection={"column"} justifyContent="flex-start">
+          <Text variant={["xs", "md", "md"]}>{name}</Text>
+          <Text variant={["lg", "xl", "xl"]}>{value}</Text>
+        </Flex>
+      </Column>
+    )
+  }
```

Comment:
**non-blocking:**  If I remember correctly, there was a discussion about no longer using this pattern in Eigen (sorry, I can't find it anymore) and doing it the React way by creating a functional component outside of this component. I think the arguments were:
- The JSX template part is easier to read if components and function calls are not mixed.
- (Probably) performance improvements because React is optimized for it
- no closure and therefor easier to debug.

What do you think?

Reply:
That's true, I remember the conversation we had and how we decided it 😄 I will update 👍